data/tutorials/getting-started/1_02_your_first_ocaml_program.md

```diff
@@ -78,10 +72,10 @@ $ tree
 └── test
     ├── dune
     └── hello.ml
-
-4 directories, 8 files
 ```
 
+Unlike in Unix where they contain compiled binaries, directories `lib` and `bin` contain source code files, for libraries and programs, respectively. This is the a convention used in many OCaml projects, including those created by Dune. All the built artifacts, and a copy of the sources, are stored in the `_build` directory. You shall not edit anything in the `_build` directory.
```

Comment:
```suggestion
Unlike in Unix where they contain compiled binaries, directories `lib` and `bin` contain source code files, for libraries and programs, respectively. This is the a convention used in many OCaml projects, including those created by Dune. All the built artifacts, and a copy of the sources, are stored in the `_build` directory. It does not make sense to edit anything in the `_build` directory, since any manual edits will be overwritten during subsequent builds.
```